src/overlays/actors/ovl_En_Wonder_Talk/z_en_wonder_talk.h

```diff
@@ -6,9 +6,21 @@
 
 struct EnWonderTalk;
 
+typedef void (*EnWonderTalkFunc)(struct EnWonderTalk*, GlobalContext*);
+
 typedef struct EnWonderTalk {
     /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0x1C];
+    /* 0x014C */ EnWonderTalkFunc actionFunc;
+    /* 0x0150 */ s16 unk_150;
+    /* 0x0152 */ s16 unk_152;
+    /* 0x0154 */ s16 switchFlag;
+    /* 0x0156 */ s16 unk_156;
+    /* 0x0158 */ s16 unk_158;
+    /* 0x0160 */ s16 unk_15A;
+    /* 0x015C */ f32 unk_15C;
+    /* 0x0160 */ f32 fHeight;
```

Comment:
I don't think the fact that it's a float needs to be part of the name for this struct member.
```suggestion
    /* 0x0160 */ f32 height;
```